src/Raven.Server/Smuggler/Documents/Handlers/SmugglerHandler.cs

```diff
@@ -482,6 +482,7 @@ private void ImportDocumentsFromCsvStream(Stream stream, DocumentsOperationConte
             if (string.IsNullOrEmpty(entity) == false && char.IsLower(entity[0]))
                 entity = char.ToUpper(entity[0]) + entity.Substring(1);
 
+            result.AddMessage($"Import collection:{entity}");
```

Comment:
please don't use addmessage, use addInfo instead - as it appends timestamp as well.